packages/tasks/src/tasks/text-to-audio/inference.ts

```diff
@@ -0,0 +1,58 @@
+/**
+ * Inference code generated from the JSON schema spec in ./spec
+ *
+ * Using src/scripts/inference-codegen
+ */
+/**
+ * Inputs for Text To Audio inference
+ */
+export interface TextToAudioInput {
+	/**
+	 * The input text data
+	 */
+	data: string;
+	/**
+	 * Additional inference parameters
+	 */
+	parameters?: TextToAudioParameters;
+	[property: string]: unknown;
+}
+/**
+ * Additional inference parameters
+ *
+ * Additional inference parameters for Text To Audio
+ */
+export interface TextToAudioParameters {
+	/**
+	 * Parametrization of the text generation process
+	 */
+	generate?: GenerationParameters;
```

Comment:
We should also support forward params so we can pass things such as `speaker_embeddings` in SpeechT5 https://huggingface.co/microsoft/speecht5_tts

Reply:
Let's do it as a follow-up